test/memo_test.rb

```diff
@@ -0,0 +1,166 @@
+# frozen_string_literal: true
+
+require 'minitest/autorun'
+require 'rack/test'
+require_relative 'constants'
+require_relative '../memo'
+
+class MemoTest < Minitest::Test
+  include Rack::Test::Methods
+
+  def app
+    Sinatra::Application
+  end
+
+  def setup
+    File.delete(TEST_DB_PATH) if File.exist?(TEST_DB_PATH)
+    memos = {
+      "test-uuid-1": {
+        "public_id": 'test-uuid-1',
+        "title": '買い物リスト',
+        "content": 'トマトジュース\nティッシュペーパー'
```

Comment:
```suggestion
        "content": "トマトジュース\nティッシュペーパー"
```

にしないと`\n`はただの文字列として扱われます

Reply:
c6c38cf にて対応しました🙇‍♂️